commands.txt

```diff
@@ -0,0 +1,51 @@
+## Commands
+This section lists command(s) run by hotspotFingerprintsCollector workflow
+
+* Running hotspotFingerprintsCollector
+
+=== Commands for downsampling, markDuplicates and generateFingerprint ===.
+
+<<<
+ set -euo pipefail
+ 
+ seqtk sample -s 100 ~{fastqR1} ~{maxReads} > ~{fastqR1m}
+ gzip ~{fastqR1m}
+ 
+ seqtk sample -s 100 ~{fastqR2} ~{maxReads} > ~{fastqR2m}
+ gzip ~{fastqR2m}
+>>>
+<<<
+  set -euo pipefail
+  $GATK_ROOT/bin/gatk --java-options "-Xmx~{jobMemory - overhead}G" MarkDuplicates \
```

Comment:
For all sections: this README can be improved if commands are formatted so that they look more 'user-friendly'. A good example would be https://github.com/oicr-gsi/delly/blob/master/README.md